meta/src/html.rs

```diff
@@ -60,9 +67,11 @@ impl std::fmt::Debug for HtmlContext {
 /// fn MyApp(cx: Scope) -> impl IntoView {
 ///     provide_meta_context(cx);
 ///
+///     let attrs = vec![(String::from("data-theme"), TextProp::from("dark"))];
+///
 ///     view! { cx,
 ///       <main>
-///         <Html lang="he" dir="rtl"/>
+///         <Html lang="he" dir="rtl" attributes=attrs/>
 ///       </main>
```

Comment:
I think there is a way to implement the proper `From` trait for the `attrs` vector such one can use:
`let attrs = vec![("data-theme", "dark")];`
But I did not manage to solve that.